deploy/olm/openshift/0.7.1/dynatrace-monitoring.v0.7.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,439 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "dynatrace.com/v1alpha1",
+          "kind": "OneAgent",
+          "metadata": {
+            "name": "oneagent",
+            "namespace": "dynatrace"
+          },
+          "spec": {
+            "apiUrl": "https://ENVIRONMENTID.live.dynatrace.com/api",
+            "args": [
+              "APP_LOG_CONTENT_ACCESS=1"
+            ],
+            "env": [],
+            "image": "registry.connect.redhat.com/dynatrace/oneagent",
```

Comment:
Technically this shouldn't be needed anymore since `RELATED_IMAGE_DYNATRACE_ONEAGENT` has a different default already but leave it for now.